packages/protocol/test-sol/stability/Reserve.t.sol

```diff
@@ -313,6 +360,45 @@ contract ReserveTest_transfers is ReserveTest {
     reserve.transferGold(otherReserveAddress, amount);
   }
 
+  function test_transferErc20Token() public {
```

Comment:
Same as the comment above. Split this down into separate functions for each test scenario and give the functions descriptive names.